src/lib/autocomplete/autocomplete.spec.ts

```diff
@@ -1350,50 +1381,49 @@ describe('MatAutocomplete', () => {
       trigger = fixture.componentInstance.trigger;
       closingActionSpy = jasmine.createSpy('closing action listener');
       closingActionsSub = trigger.panelClosingActions.subscribe(closingActionSpy);
-    });
+
+      fixture.whenStable().then(() => {
+        expect(closingActionSpy).not.toHaveBeenCalled();
```

Comment:
In general we try not to do expectations inside `beforeEach`. Can you move it into its own test?

Reply:
Done.